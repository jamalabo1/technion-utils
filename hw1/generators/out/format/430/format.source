D Y 876 1
A W 293 2
C Y 276 3
C W 277 4
D X 347 5
A X 797 6
A X 128 7
A Z 325 8
C Z 108 9
B W 681 10
C Z 957 11
D W 285 12
A X 369 13
B Y 410 14
B Y 794 15
C W 283 16
A Z 508 17
C W 995 18
D W 108 19
B Z 363 20
B W 258 21
D W 299 22
C Z 257 23
B X 454 24
A W 630 25
B W 683 26
B X 48 27
A Y 488 28
D Z 649 29
C W 231 30
A X 785 31
A W 705 32
B W 822 33
B X 709 34
D Y 421 35
A Y 765 36
A X 572 37
A Y 33 38
C X 481 39
C Y 895 40
A Y 322 41
C Y 179 42
C Y 139 43
C Z 517 44
C Y 726 45
D Z 874 46
A X 177 47
D X 206 48
A Z 773 49
A Y 984 50
D Y 908 51
D Y 287 52
D Z 46 53