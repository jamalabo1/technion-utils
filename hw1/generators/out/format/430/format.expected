Blockchain Info:
1
Sender Name: D
Receiver Name: Z
Transaction Value: 46
Transaction timestamp: 53
2
Sender Name: D
Receiver Name: Y
Transaction Value: 287
Transaction timestamp: 52
3
Sender Name: D
Receiver Name: Y
Transaction Value: 908
Transaction timestamp: 51
4
Sender Name: A
Receiver Name: Y
Transaction Value: 984
Transaction timestamp: 50
5
Sender Name: A
Receiver Name: Z
Transaction Value: 773
Transaction timestamp: 49
6
Sender Name: D
Receiver Name: X
Transaction Value: 206
Transaction timestamp: 48
7
Sender Name: A
Receiver Name: X
Transaction Value: 177
Transaction timestamp: 47
8
Sender Name: D
Receiver Name: Z
Transaction Value: 874
Transaction timestamp: 46
9
Sender Name: C
Receiver Name: Y
Transaction Value: 726
Transaction timestamp: 45
10
Sender Name: C
Receiver Name: Z
Transaction Value: 517
Transaction timestamp: 44
11
Sender Name: C
Receiver Name: Y
Transaction Value: 139
Transaction timestamp: 43
12
Sender Name: C
Receiver Name: Y
Transaction Value: 179
Transaction timestamp: 42
13
Sender Name: A
Receiver Name: Y
Transaction Value: 322
Transaction timestamp: 41
14
Sender Name: C
Receiver Name: Y
Transaction Value: 895
Transaction timestamp: 40
15
Sender Name: C
Receiver Name: X
Transaction Value: 481
Transaction timestamp: 39
16
Sender Name: A
Receiver Name: Y
Transaction Value: 33
Transaction timestamp: 38
17
Sender Name: A
Receiver Name: X
Transaction Value: 572
Transaction timestamp: 37
18
Sender Name: A
Receiver Name: Y
Transaction Value: 765
Transaction timestamp: 36
19
Sender Name: D
Receiver Name: Y
Transaction Value: 421
Transaction timestamp: 35
20
Sender Name: B
Receiver Name: X
Transaction Value: 709
Transaction timestamp: 34
21
Sender Name: B
Receiver Name: W
Transaction Value: 822
Transaction timestamp: 33
22
Sender Name: A
Receiver Name: W
Transaction Value: 705
Transaction timestamp: 32
23
Sender Name: A
Receiver Name: X
Transaction Value: 785
Transaction timestamp: 31
24
Sender Name: C
Receiver Name: W
Transaction Value: 231
Transaction timestamp: 30
25
Sender Name: D
Receiver Name: Z
Transaction Value: 649
Transaction timestamp: 29
26
Sender Name: A
Receiver Name: Y
Transaction Value: 488
Transaction timestamp: 28
27
Sender Name: B
Receiver Name: X
Transaction Value: 48
Transaction timestamp: 27
28
Sender Name: B
Receiver Name: W
Transaction Value: 683
Transaction timestamp: 26
29
Sender Name: A
Receiver Name: W
Transaction Value: 630
Transaction timestamp: 25
30
Sender Name: B
Receiver Name: X
Transaction Value: 454
Transaction timestamp: 24
31
Sender Name: C
Receiver Name: Z
Transaction Value: 257
Transaction timestamp: 23
32
Sender Name: D
Receiver Name: W
Transaction Value: 299
Transaction timestamp: 22
33
Sender Name: B
Receiver Name: W
Transaction Value: 258
Transaction timestamp: 21
34
Sender Name: B
Receiver Name: Z
Transaction Value: 363
Transaction timestamp: 20
35
Sender Name: D
Receiver Name: W
Transaction Value: 108
Transaction timestamp: 19
36
Sender Name: C
Receiver Name: W
Transaction Value: 995
Transaction timestamp: 18
37
Sender Name: A
Receiver Name: Z
Transaction Value: 508
Transaction timestamp: 17
38
Sender Name: C
Receiver Name: W
Transaction Value: 283
Transaction timestamp: 16
39
Sender Name: B
Receiver Name: Y
Transaction Value: 794
Transaction timestamp: 15
40
Sender Name: B
Receiver Name: Y
Transaction Value: 410
Transaction timestamp: 14
41
Sender Name: A
Receiver Name: X
Transaction Value: 369
Transaction timestamp: 13
42
Sender Name: D
Receiver Name: W
Transaction Value: 285
Transaction timestamp: 12
43
Sender Name: C
Receiver Name: Z
Transaction Value: 957
Transaction timestamp: 11
44
Sender Name: B
Receiver Name: W
Transaction Value: 681
Transaction timestamp: 10
45
Sender Name: C
Receiver Name: Z
Transaction Value: 108
Transaction timestamp: 9
46
Sender Name: A
Receiver Name: Z
Transaction Value: 325
Transaction timestamp: 8
47
Sender Name: A
Receiver Name: X
Transaction Value: 128
Transaction timestamp: 7
48
Sender Name: A
Receiver Name: X
Transaction Value: 797
Transaction timestamp: 6
49
Sender Name: D
Receiver Name: X
Transaction Value: 347
Transaction timestamp: 5
50
Sender Name: C
Receiver Name: W
Transaction Value: 277
Transaction timestamp: 4
51
Sender Name: C
Receiver Name: Y
Transaction Value: 276
Transaction timestamp: 3
52
Sender Name: A
Receiver Name: W
Transaction Value: 293
Transaction timestamp: 2
53
Sender Name: D
Receiver Name: Y
Transaction Value: 876
Transaction timestamp: 1